src/mainwindow.ui

```diff
@@ -202,6 +202,15 @@
      <addaction name="actionPreview540"/>
      <addaction name="actionPreview720"/>
     </widget>
+    <widget class="QMenu" name="menuTimeFormat">
+     <property name="title">
+      <string>Time Format</string>
+     </property>
+     <addaction name="actionTimeFrames"/>
+     <addaction name="actionTimeClock"/>
```

Comment:
Please move "Time Format" to just below "Timeline" in the Settings menu. And move "Frames" to the bottom of the sub-menu.

Reply:
Done